tests/phpunit/CRM/Event/Form/Registration/ConfirmTest.php

```diff
@@ -721,4 +721,37 @@ protected function submitPaidEvent(array $submitValues = []): void {
     ], $submitValues));
   }
 
+  public function testRegistrationWithoutCiviContributeEnabled(): void {
+    $mut = new CiviMailUtils($this, TRUE);
+    $event = $this->eventCreateUnpaid([
+      'has_waitlist' => 1,
+      'max_participants' => 1,
+      'start_date' => 20351021,
+      'end_date' => 20351023,
+      'registration_end_date' => 20351015,
+    ]);
+    CRM_Core_BAO_ConfigSetting::disableComponent('CiviContribute');
```

Comment:
If I moved this before eventCreteUnpaid I ran into a whole bunch of issues with Example Data creation so decided to skip overall of that for now